twitter_post_block.py

```diff
@@ -69,9 +63,12 @@ def _post_tweet(self, payload):
 
         status = response.status_code
         if status != 200:
+            response = response.json()
             self._logger.error(
```

Comment:
In this case, I disagree. I think that you'd always want to know why the post failed, since usually it's something you could fix in block configuration. In general, though, I agree, error is probably used to widely.